content/posts/2021/reana-client-python-api.md

```diff
@@ -0,0 +1,223 @@
+---
+title: "How to run workflows using reana-client Python API"
+date: 2021-02-22T15:00:00+01:00
+tags:
+  - API
+  - client
+  - Python
+---
+
+Here you can find a step by step guide on how to submit workflows to
+REANA using the `reana-client` Python API. This is a simple example,
+for illustrative purposes, using the [Serial](http://docs.reana.io/running-workflows/supported-systems/serial/)
+workflow engine and a simple [ROOT6 / RooFit analysis](https://github.com/reanahub/reana-demo-root6-roofit/)
+example. We will create a new workflow, run it and retrieve its results
+using the `reana-client` Python API.
+<!--more-->
+
+### Prerequisite
+
+The only prerequisite to follow this guide is to install `reana-client`, if you
+haven't done that already:
+```
+$ pip install reana-client==0.7.2
+```
+
+### Configurations
```

Comment:
Configuring access

That might be more specific, since "configurations" sounds a bit too generic.

We can actually turn all the headings into this form;

- Installing prerequisites
- Configuring access
- Specifying workflow
- Creating workflow
- Uploading files
- Starting workflow
- Checking workflow status
- Listing workspace files
- Downloading outputs